src/components/Modal/Modal.tsx

```diff
@@ -1,26 +1,34 @@
 import { Modal as AntdModal } from "antd";
+import { Content } from "antd/es/layout/layout";
+import classNames from "classnames";
 import React, { forwardRef, useImperativeHandle, useState } from "react";
-import { Divider } from "../..";
+import { Button, ChaiiText, Divider } from "../..";
 import "../../sharedStyles.scss";
+import styles from "./modal.module.scss";
 
 interface Props {
   heading?: string;
   ref?: any;
   onClose?: () => void;
```

Comment:
I think `onClose` and `onOpen` should not be optional props.

I understand we are not introducing these in this change, but we should look into this.